Umbraco-Uno/Uno-pedia/Settings/Custom-Code/index.md

```diff
@@ -0,0 +1,29 @@
+---
+versionFrom: 8.0.0
+---
+
+# Custom Code
+
+In Umbraco Uno, it is possible to enable Custom Code, if you are on a Standard or Professional plan.
+The way you activate this feature is very straight forward:
+
+1. Navigate to your project in the portal
+
+2. Go to settings
+
+3. Select ***Enable Custom Code***
+
+![this image shows what the menu looks like](images/Enable-Custom-Code.png)
+
+4. You will be taken to a new page with a disclaimer, telling you that once you enable custom code you cannot turn it off again.
```

Comment:
4. You will be taken to a new page with a disclaimer, telling you that once you enable custom code you cannot revert that change.